src/grammar.coffee

```diff
@@ -268,6 +271,7 @@ grammar =
   Param: [
     o 'ParamVar',                               -> new Param $1
     o 'ParamVar ...',                           -> new Param $1, null, on
+    o '... ParamVar',                           -> new Param $2, null, on    
```

Comment:
Allowing the dots on either side was my idea: https://github.com/coffeescript6/discuss/issues/85

We definitely can’t get rid of the dots-on-the-right syntax. One of the goals of CS2 is to introduce as few breaking changes as possible, and forcing the dots to always be on the left is a gratuitous breaking change. I suggested allowing on either side, because I and others have had the experience of going back and forth with ES and typing the dots on the wrong side; and it seemed like an easy enough quick win to just allow them on either side. There’s already precedent in CoffeeScript for such a thing, in that we allow `&&` as well as `and`, `||` as well as `or`, etc.

I think CoffeeScript recommended style will remain that the dots _should_ go on the right, just as you shouldn’t type `&&` in CoffeeScript, but I think this is one of those cases where the compiler can be flexible enough to accept either style.

If you disagree let’s please discuss it in https://github.com/coffeescript6/discuss/issues/85. @jashkenas, @lydell, your opinions would be welcomed.